### PR TITLE
Check for mysql root access.

### PR DESCRIPTION
**Pull request by a-martynovich**
wottsecurity/api#456

### CODE REVIEW

File: agent/security_helper.py [hunk 1]
```diff
@@ -249,3 +250,17 @@ def audit_sshd():
         if insecure_params[param] != config[param][1]:
             issues[param] = insecure_params[param]
     return issues
+
+
+def mysql_root_access():
+    try:
+        subprocess.check_call(["mysql", "-uroot", "-eSHOW DATABASES;"], timeout=5,
+                              stdout=subprocess.DEVNULL, stderr=subprocess.DEVNULL)
+        return True
```
Comment (rptrchv) [inline agent/security_helper.py]: pls move this out of the `try` clause

Comment (a-martynovich) [reply agent/security_helper.py]: Why? What's going to change?

Comment (rptrchv) [reply agent/security_helper.py]: In this particular case - nothing besides readability. But in general a good practice is to not put in `try` anything besides the code supposed to throw an exception in some condition

Comment (a-martynovich) [reply agent/security_helper.py]: Splitting these two lines into two blocks, one of which will be at the bottom, will decrease readability. For no reason whatsoever.

Comment (rptrchv) [reply agent/security_helper.py]: I don't agree, but let's forget about subjective readability. What about following good practice rules?

**Pull request merged**